sdk/iot/hub/inc/az_iot_hub_client.h

```diff
@@ -89,14 +89,17 @@ AZ_NODISCARD az_result az_iot_hub_client_init(
  * {iothubhostname}/{device_id}/{module_id}/?api-version=2018-06-30&{user_agent}
  *
  * @param[in] client The #az_iot_hub_client to use for this call.
- * @param[in] mqtt_user_name An empty #az_span with sufficient capacity to hold the MQTT user name.
- * @param[out] out_mqtt_user_name The output #az_span containing the MQTT user name.
+ * @param[out] mqtt_user_name A char buffer with sufficient capacity to hold the MQTT user name.

```

Comment:
Please keep exactly the same format as in Provisioning for documentation. That way, if (when) we decide to change, it'll be simpler to just use a reg-ex.

Reply:
I will wait till DPS is merged since some of the wording still seems to be in flux. But will be updated.

Reply:
updated